lux/executor/PandasExecutor.py

```diff
@@ -156,20 +156,28 @@ def execute_aggregate(vis: Vis, isFiltered=True):
             if measure_attr.attribute == "Record":
                 vis._vis_data = vis.data.reset_index()
                 # if color is specified, need to group by groupby_attr and color_attr
-
                 if has_color:
                     vis._vis_data = (
                         vis.data.groupby([groupby_attr.attribute, color_attr.attribute], dropna=False)
                         .count()
                         .reset_index()
                     )
-                    vis._vis_data = vis.data.rename(columns={"index": "Record"})
+                    index_name = list(
+                        filter(
+                            lambda k: groupby_attr.attribute not in k and color_attr.attribute not in k,
```

Comment:
Why do we do `groupby_attr.attribute not in k` does that check if the groupby attribute is in the column name `k`? Why not just directly check if they're equal?

Reply:
Yeah here I'm trying to get the name of the column with the aggregate counts, which is named either "index" or whatever column a user has set to be the index. Ideally I would like to get the index name from the original Lux DF using something like `ldf.index.name` but the PandasExecutor doesn't have a reference to this DF, so we'd have to pass down the index name some other way. Instead of adding another parameter to the global config just for this, I just filtered out the groupby and color attributes to get at the index column.

Reply:
I think this makes sense but is a bit messy. We should look into whether there is a way of rewriting the previous line (possibly by replacing `.count` via `.transform` or `.agg` instead, or something via the reset_index) so that we can explicitly do the renaming to `Record` here. Something like [this](https://stackoverflow.com/questions/40914200/can-i-assign-a-reset-index-a-name) might work.
```python
vis._vis_data = (
                        vis.data.groupby([groupby_attr.attribute, color_attr.attribute], dropna=False)
                        .count()
                        .reset_index()
                    )
```

Reply:
My concern for the `index_name = list( filter(lambda k: groupby_attr.attribute not in k, vis.data.columns)[0])` is that I'm not sure if there would ever be an edge case where there is no matching filter and this causes an IndexError. So if we can combine it with the previous statement, it would be better.

Reply:
Thanks for the rename suggestion, that solution seems to work out. I also found that I vis.data still has the index name so I can use that instead of filtering out the non-index columns.